src/main/rx/rx.c

```diff
@@ -228,24 +238,37 @@ bool rxIsReceivingSignal(void)
     return rxSignalReceived;
 }
 
+bool rxAreFlightChannelsValid(void)
```

Comment:
Yeah, i was finding it hard to name it well.  Yes, should be false if any of the 4 flight channels are invalid for whayever reason, including disconnected PWM cable or out of range stick.